Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:bionic
+
+RUN useradd -m retdec
+WORKDIR /home/retdec
+ENV HOME /home/retdec
+
+RUN apt-get -y update && \
+	DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential git bc graphviz upx cmake python zlib1g-dev flex bison libtinfo-dev autoconf pkg-config m4 libtool wget
```

Comment:
I would also use `--no-install-recommends`

Reply:
Thanks. I'll eventually split this into a multi-stage so the build tools won't need to be in the production image.